app/code/Magento/QuoteGraphQl/Model/Cart/SetShippingAddressesOnCart.php

```diff
@@ -51,7 +51,9 @@ public function execute(ContextInterface $context, CartInterface $cart, array $s
         $shippingAddressInput = current($shippingAddressesInput) ?? [];
         $customerAddressId = $shippingAddressInput['customer_address_id'] ?? null;
 
-        if (!$customerAddressId && !isset($shippingAddressInput['address']['save_in_address_book'])) {
+        if (!$customerAddressId &&
+            isset($shippingAddressInput['address']) &&
+            !isset($shippingAddressInput['address']['save_in_address_book'])) {
```

Comment:
Hello @atwixfirster,
could you please fix the minor styling issue of this `if` statement to reflect the PSR-12 coding style guide?

> Expressions in parentheses MAY be split across multiple lines, where each subsequent line is indented at least once. When doing so, the first condition MUST be on the next line. The closing parenthesis and opening brace MUST be placed together on their own line with one space between them. Boolean operators between conditions MUST always be at the beginning or at the end of the line, not a mix of both.

Reply:
**done**